src/main/webapp/app/tasks/task-search/task-search.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { TaskSearchComponent } from './task-search.component';
+
+describe('TaskSearchComponent', () => {
```

Comment:
I would expect actual added test cases and not only the bare spec file that is generated by ng generate for instance